hera_mc/cm_health.py

```diff
@@ -0,0 +1,133 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2018 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Functions to handle various cm table health checks.
```

Comment:
For terminology, I guess the idea of a "health check" implies to me that this code is going to query the various bits of hardware in the running system and check that they're all operating correctly. What the code actually does I might describe more as a "consistency check" or something. But it would be annoying to switch the terminology everywhere and I don't think it's worth doing unless you were already dissatisfied with it.